pkg/vpc/vpc.go

```diff
@@ -31,10 +31,16 @@ func SetSubnets(spec *api.ClusterConfig) error {
 		cidr := api.DefaultCIDR()
 		vpc.CIDR = &cidr
 	}
-	prefix, _ := spec.VPC.CIDR.Mask.Size()
-	if (prefix < 16) || (prefix > 24) {
-		return fmt.Errorf("VPC CIDR prefix must be betwee /16 and /24")
+
+	cidrs := append(spec.VPC.ExtraCIDRs, vpc.CIDR)
+	for _, cidr := range cidrs {
+		prefix, _ := cidr.Mask.Size()
+		if (prefix < 16) || (prefix > 24) {
+			return fmt.Errorf("VPC CIDR (%v) prefix must be between /16 and /24", cidr)
```

Comment:
Should this happen during the validation instead?

Reply:
This slipped in from #773.